LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2024 The Simons Foundation, Inc. - All Rights Reserved.
+   Copyright 2025 The Simons Foundation, Inc. - All Rights Reserved.
```

Comment:
```suggestion
   Copyright 2024 The Simons Foundation, Inc. - All Rights Reserved.
```
Not really sure how these things work but I assume the year should refer to when the license was added to the repository.